client/package-lock.json

```diff
@@ -1,131 +0,0 @@
-{
```

Comment:
The existence of this caused issues for me during a fresh `yarn install` and it isn't seemingly required when running `yarn install` from the project root due to the workspace configuration.

But let me know if I should add it back.